anaconda/Dockerfile

```diff
@@ -1,32 +1,74 @@
-FROM debian:latest
-
-#  $ docker build . -t continuumio/anaconda:latest -t continuumio/anaconda:2018.12 -t continuumio/anaconda2:latest -t continuumio/anaconda2:2018.12
-#  $ docker run --rm -it continuumio/anaconda2:latest /bin/bash
-#  $ docker push continuumio/anaconda:latest
-#  $ docker push continuumio/anaconda:2018.12
-#  $ docker push continuumio/anaconda2:latest
-#  $ docker push continuumio/anaconda2:2018.12
+FROM alpine:3.9 as alpine-glibc
 
+LABEL MAINTAINER="Vlad Frolov"
+LABEL SRC=https://github.com/frol/docker-alpine-glibc
 ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
-ENV PATH /opt/conda/bin:$PATH
 
-RUN apt-get update --fix-missing && apt-get install -y wget bzip2 ca-certificates \
-    libglib2.0-0 libxext6 libsm6 libxrender1 \
-    git mercurial subversion
+# Here we install GNU libc (aka glibc) and set C.UTF-8 locale as default.
+RUN ALPINE_GLIBC_BASE_URL="https://github.com/sgerrand/alpine-pkg-glibc/releases/download" && \
```

Comment:
Are we really ok using someone else's libc?  Isn't this something we should be building ourselves?

Reply:
We could, do we have a version for this? This is also close to the example you sent me.